integration-tests/smoke/ccip/ccip_migration_to_v_1_6_test.go

```diff
@@ -40,6 +45,355 @@ var (
 	onRampABI        = abihelpers.MustParseABI(onramp.OnRampABI)
 )
 
+// TestV1_5_Message_RMNRemote this test verify that 1.5 lane can send message when using RMNRemote
+func TestV1_5_Message_RMNRemote(t *testing.T) {
```

Comment:
should it be moved to e2e_test - https://github.com/smartcontractkit/chainlink/blob/develop/deployment/ccip/changeset/v1_5/e2e_test.go? does it need to be executed with docker?